packages/web/src/scss/components/UNSTABLE_Divider/_UNSTABLE_Divider.scss

```diff
@@ -0,0 +1,8 @@
+@use 'theme' as theme;
+
+.UNSTABLE_Divider {
+    display: block;
```

Comment:
```suggestion
```

This is already provided by user agent:

![image](https://github.com/lmc-eu/spirit-design-system/assets/5614085/8c2fdfe4-f167-48bb-9b15-57f00d888176)
![image](https://github.com/lmc-eu/spirit-design-system/assets/5614085/de1c13d9-7ffe-418a-8802-f0986feaa653)
![image](https://github.com/lmc-eu/spirit-design-system/assets/5614085/af0b9b46-87a8-4132-b4b6-5cd75691b2a2)